bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -101,9 +104,12 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     public void getTibberParameters() {
         String response = "";
+        versionString = FrameworkUtil.getBundle(this.getClass()).getVersion().toString();
```

Comment:
Can this be initialized in `initialize()`? This would make method `open()` independent of this method in terms of order of initialization.

Reply:
Good point, moved to `initialize()`.